KIPs/kip-fungible_token.md

```diff
@@ -157,6 +160,43 @@ Returns the amount which `_spender` is still allowed to withdraw from `_owner`.
 function allowance(address _owner, address _spender) public view returns (uint256 remaining)
 ```
 
+#### mint
+
+Mints `_value` amount of tokens to address `_to`, and MUST fire the [Transfer event](#transfer-1).
+The function SHOULD `throw` if the message caller's balance does not have enough tokens to spend.
+
+*Note* Function Modifiers can be used for `mint` function on their own purpose.
+
+```solidity
+function mint(address account, uint256 amount) public onlyMinter returns (bool) {
+```
+
+#### burn
+
+Burns `_value` amount of tokens and MUST fire the [Transfer event](#transfer-1).
+The function SHOULD `throw` if the `_to` is `0x0`.
```

Comment:
```suggestion
```
This sentence doesn't seem to be about the burn function. It might be better to insert another sentence.